src/components/atoms/TextArea/TextArea.component.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useMemo } from 'react'
+import { Input as AntInput } from 'antd'
+import { TextAreaProps as AntTextAreaProps } from 'antd/lib/input/TextArea'
+
+import { CSSProperties } from 'styled-components'
+
+import { Text } from '@components/atoms/Text/Text.component'
+
+import { PreviewProps } from '@src/types/preview'
+
+import { ComponentProps } from '@src/types/component'
+
+import VisibilityComponent from '@components/common/ResponsiveVisibility/ResponsiveVisibility.component'
+
+import { getClasses, isStyleObject } from '@src/utils/common'
+
+import styles from './TextArea.module.css'
+
+export interface TextAreaProps
+  extends PreviewProps,
+    ComponentProps<Omit<AntTextAreaProps, 'style'>> {
+  style?: Record<string, unknown> | string
+  inputStyle?: Record<string, unknown> | string
+  placeholderStyle?: Record<string, unknown> | string
+  placeholder?: string
+  errorMessage?: string
+}
+
+const TextArea = React.forwardRef<HTMLDivElement, TextAreaProps>((props, ref) => {
+  const {
+    style = {},
+    inputStyle = {},
```

Comment:
will it cause confusing when it is style but can be used as classname? 🤔

Reply:
@quy-truong it because the code generator will generate it as module css.
when pass to props it will be
```
import styles from './styles.modules.css'
<Input inputStyle={styles.input_name_x_input}
```
i'm not sure about best solution right now, but i think we need improve it later